tests/data_context/store/test_ge_cloud_store_backend.py

```diff
@@ -22,393 +6,25 @@
-                                ("run_name_template", None),
-                                ("expectation_suite_name", None),
-                                ("batch_request", {}),
-                                ("action_list", []),
-                                ("evaluation_parameters", {}),
-                                ("runtime_configuration", {}),
-                                ("validations", []),
-                                ("profilers", []),
-                                ("ge_cloud_id", None),
-                                ("expectation_suite_ge_cloud_id", None),
-                            ]
-                        ),
-                    },
-                }
-            },
-        )
-
-
-@pytest.mark.cloud
-@pytest.mark.unit
-def test_list_keys(
-    construct_ge_cloud_store_backend: Callable[
-        [GXCloudRESTResource], GeCloudStoreBackend
-    ],
-) -> None:
-    store_backend = construct_ge_cloud_store_backend(GXCloudRESTResource.CHECKPOINT)
-
-    with mock.patch("requests.Session.get", autospec=True) as mock_get:
-        store_backend.list_keys()
-        mock_get.assert_called_with(
-            mock.ANY,  # requests.Session object
-            f"{CLOUD_DEFAULT_BASE_URL}organizations/51379b8b-86d3-4fe7-84e9-e1a52f4a414c/checkpoints",
-        )
-
-
-@pytest.mark.cloud
-@pytest.mark.unit
-def test_remove_key(
-    construct_ge_cloud_store_backend: Callable[
-        [GXCloudRESTResource], GeCloudStoreBackend
-    ],
+def test_ge_cloud_store_backend_init_raises_deprecation_warning(
+    ge_cloud_access_token: str,
 ) -> None:
-    store_backend = construct_ge_cloud_store_backend(GXCloudRESTResource.CHECKPOINT)
-
-    with mock.patch("requests.Session.delete", autospec=True) as mock_delete:
-        mock_response = mock_delete.return_value
-        mock_response.status_code = 200
+    ge_cloud_base_url = CLOUD_DEFAULT_BASE_URL
+    ge_cloud_credentials = {
+        "access_token": ge_cloud_access_token,
+        "organization_id": "51379b8b-86d3-4fe7-84e9-e1a52f4a414c",
+    }
 
-        store_backend.remove_key(
-            (
-                "checkpoint",
-                "0ccac18e-7631-4bdd-8a42-3c35cce574c6",
-            )
-        )
-        mock_delete.assert_called_with(
-            mock.ANY,  # requests.Session object
-            f"{CLOUD_DEFAULT_BASE_URL}organizations/51379b8b-86d3-4fe7-84e9-e1a52f4a414c/checkpoints/0ccac18e-7631"
-            "-4bdd"
-            "-8a42-3c35cce574c6",
-            json={
-                "data": {
-                    "type": "checkpoint",
-                    "id": "0ccac18e-7631-4bdd-8a42-3c35cce574c6",
-                    "attributes": {"deleted": True},
-                }
-            },
+    with pytest.deprecated_call() as depr:
+        GeCloudStoreBackend(
+            ge_cloud_base_url=ge_cloud_base_url,
+            ge_cloud_credentials=ge_cloud_credentials,
+            ge_cloud_resource_type=GXCloudRESTResource.CHECKPOINT,
```

Comment:
```suggestion
    with pytest.warns(DeprecationWarning):
        GeCloudStoreBackend(
            ge_cloud_base_url=ge_cloud_base_url,
            ge_cloud_credentials=ge_cloud_credentials,
            ge_cloud_resource_type=GXCloudRESTResource.CHECKPOINT,
```

If you do this you shouldn't need the second assert.

https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html#warns

Reply:
so I think I actually changed this right as you were reviewing - sorry about that!

I opted to use `pytest.deprecated_call()` which is a shorthand for `warns(DeprecationWarning)`.

Reply:
Why do you need the extra assert on on line `30`?